build/schema.graphql

```diff
@@ -98,4 +18,7 @@ type activePool @entity {
     ETH: BigDecimal!
     LUSDDebt: BigDecimal!
     totalEthSent: BigDecimal
+    currentOwner: String
```

Comment:
The field name styles are pretty inconsistent. You're using camel case here `currentOwner`, snakecase in the `ETHSent` entitity with `usd_value`, and some hybrid with `LUSDDebt`. Cleanup the style and maybe name stuff more appropriately.